src/main.js

```diff
@@ -21,10 +24,21 @@ new Vue({
   store,
   vuetify,
   created() {
+    const buslines = localStorage.getItem("buslines");
+    if (buslines) {
+      const buslinesData = JSON.parse(buslines);
+      this.$store.commit("saveBuslines", buslinesData);
+    }
     const userDetails = localStorage.getItem("user");
+    const subscribedBuslines = localStorage.getItem("subscribedBuslines");
+    const unsubscribedBuslines = localStorage.getItem("unsubscribedBuslines");
     if (userDetails) {
       const userData = JSON.parse(userDetails);
+      const subscribesData = JSON.parse(subscribedBuslines);
+      const unsubscribesData = JSON.parse(unsubscribedBuslines);
       this.$store.commit("setUserData", userData);
+      this.$store.commit("saveSubscribes", subscribesData);
+      this.$store.commit("saveUnsubscribes", unsubscribesData);
```

Comment:
maybe better place to do it is in App.vue?

Reply:
You are right! there might be better place to save it.